src/platform/forms-system/src/js/containers/FormPage.jsx

```diff
@@ -230,12 +233,30 @@ const mapDispatchToProps = {
 
 FormPage.propTypes = {
   form: PropTypes.object.isRequired,
+  appStateData: PropTypes.shape({}),
+  blockScrollOnMount: PropTypes.bool,
+  contentAfterButtons: PropTypes.element,
+  contentBeforeButtons: PropTypes.element,
+  formContext: PropTypes.shape({
+    onReviewPage: PropTypes.bool,
+  }),
+  location: PropTypes.shape({
+    pathname: PropTypes.string,
+  }),
+  params: PropTypes.shape({
+    index: PropTypes.number, // for testing only?
```

Comment:
I couldn't find where the actual page `index` was being passed into the `CustomPage` component. This `index` is only used in unit tests, as far as I could tell.